apps/mudita-center-e2e/src/specs/news/news-check-offline.e2e.ts

```diff
@@ -0,0 +1,106 @@
+import exp from "constants"
+import screenshotHelper from "../../helpers/screenshot.helper"
+import HomePage from "../../page-objects/home.page"
+import NewsPage from "../../page-objects/news.page"
+import {
+  commentsRegex,
+  linkRegex,
+  newsDateRegex,
+  newsImageRegex,
+} from "../../consts/regex-const"
+
+describe("News Page Check in Offline Mode", () => {
+  before(async () => {
+    // Switch to offline mode before starting the tests
+    await browser.setNetworkConditions({
```

Comment:
I see that the application still sees available updates during this test, and
```
const isOnline = await browser.execute(() => {
 return navigator.onLine
 })
 console.log(isOnline)
```
prints True, which means that the application is not offline

Reply:
added check with console printing for network state